docs/pages/enroll-resources/database-access/enroll-google-cloud-databases/alloydb.mdx

```diff
@@ -0,0 +1,307 @@
+   --proxy=<Var name="example.teleport.sh:443" />  \
+   --uri=alloydb://<Var name="full-instance-name" />
+```
+
+This command will generate a Teleport Database Service configuration file and
+save it to `/etc/teleport.yaml`.
+
+<Admonition title="Choose how Teleport connects to AlloyDB">
+By default, Teleport uses [_private_](https://cloud.google.com/alloydb/docs/about-private-services-access) AlloyDB endpoint. To change this to either [public](https://cloud.google.com/alloydb/docs/connect-public-ip) or [PSC](https://cloud.google.com/alloydb/docs/about-private-service-connect) endpoints, update the `endpoint_type` field:
+
+```yaml
+db_service:
+  resources:
+    - name: alloydb
+      protocol: postgres
+      uri: alloydb://projects/PROJECT/locations/REGION/clusters/CLUSTER/instances/INSTANCE
+      gcp:
+        alloydb:
+          # one of: private | public | psc (default: private)
+          endpoint_type: private
+```
+</Admonition>
+
+## Step 6/9. Configure GCP credentials
+
+The Teleport Database Service must have credentials for the
+`teleport-db-service` GCP service account.
+
+If the Teleport Database Service is hosted on a GCE instance, you can
+[change the attached service account](https://cloud.google.com/compute/docs/instances/change-service-account).
+For non-GCE deployments of Teleport, we recommend using
+[workload identity](https://cloud.google.com/iam/docs/workload-identity-federation).
+
+<details>
+  <summary>Using service account keys (insecure)</summary>
+
+Alternatively, go to that service account's Keys tab and create a new key:
+
+![Service Account Keys](../../../../img/database-access/guides/alloydb/service-account-keys.png)
+
+Make sure to choose JSON format:
+
+![Service Account New Key](../../../../img/database-access/guides/alloydb/service-account-new-key.png)
+
+Save the file. Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to
+point to the JSON credentials file you downloaded earlier. For example, if you
+use `systemd` to start `teleport`, then you should edit the service's
+`EnvironmentFile` to include the env var:
+```code
+$ echo 'GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json' | sudo tee -a /etc/default/teleport
+```
+
+<Admonition type="warning">
+A service account key can be a security risk - we only describe using a key in
+this guide for simplicity.
+We do not recommend using service account keys in production.
+See [authentication](https://cloud.google.com/docs/authentication#service-accounts)
+in the Google Cloud documentation for more information about service account
+authentication methods.
+</Admonition>
+</details>
+
+## Step 7/9. Start the Teleport Database Service
+
+(!docs/pages/includes/start-teleport.mdx service="the Teleport Database Service"!)
+
+## Step 8/9. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 9/9. Connect
+
+Once the Database Service has joined the cluster, log in to see the available
+databases:
+
+<Tabs>
```

Comment:
I would use a `Var` for the Proxy Service address instead of tabs, indicating that the user needs to assign the variable to the host and port of the Proxy Service.